### PR TITLE
Adding release notes for v1.0 to the PROOF wiki page

### PR DESCRIPTION
**Pull request by tefirman**
## Description
With the recent "v1.0" release of the overarching PROOF project, we are adding release notes to the PROOF product pages describing what has changed since it's original rollout. For more details, please visit the [PROOF umbrella repository](https://github.com/getwilds/proof/).

### CODE REVIEW

File: _datascience/proof.md [hunk 1]
```diff
@@ -26,12 +26,23 @@ primary_reviewers: vortexing, abbywall
 ## Release Notes
 See our release notes in the PROOF repo for more details: 
 
-### May 2024
+### June 2024 - PROOF v1.0
 **What's New**
+- Users can now interact with data stored in S3 buckets within WDL workflows executed in PROOF! Just make sure to have your AWS CLI credentials [established in Rhino](https://sciwiki.fredhutch.org/scicomputing/access_credentials/#configure-aws-cli) and you can use the S3 path just like any other input.
```
Comment (dtenenba) [inline _datascience/proof.md:31]: Maybe make the CLI credentials a relative link?

File: _datascience/proof.md [hunk 2]
```diff
@@ -26,12 +26,23 @@ primary_reviewers: vortexing, abbywall
...
+- GPU analysis is now possible via the `gpu` argument in your WDL task's runtime section! For an example, try running [this example script](https://github.com/getwilds/ww-test-workflows/blob/main/gpuMatrixMult/gpuMatrixMult.wdl) in the [ww-test-workflows](https://github.com/getwilds/ww-test-workflows) repo of the [DaSL WILDS](https://github.com/getwilds).
+- Additional features have been introduced to increase transparency in terms of the functionality of PROOF:
+    - New `/info` API endpoint provides details on which code base is being used.
+    - Server start date and time is now available on the "PROOF Server" tab.
+    - Documentation links have been added to the "Welcome" tab and automated server creation email for increased visibility.
 
 **Fixes**
+- The default location of the Apptainer cache directory has been moved from `scratch` to `/hpc/temp` to avoid previously reported linkage issues associated with `scratch`.
```
Comment (dtenenba) [inline _datascience/proof.md:39]: Only for users whose lab/group has a directory in /hpc/temp.

**Review by dtenenba — APPROVED** [review]
I had a couple of minor comments. Other than those, LGTM!

File: _datascience/proof.md (hunk 2 above)
Comment (tefirman) [reply _datascience/proof.md:39]: Great call again, just added a sub-bullet that describes that, i.e. defaults to home directory if the user's lab/group doesn't have /hpc/temp yet. Thanks Dan!!

Comment (dtenenba) [reply _datascience/proof.md:39]: Actually it still uses scratch if user does not have /hpc/temp yet.

**Pull request merged**